magefile.go

```diff
@@ -1969,6 +1969,11 @@ func (Integration) UpdatePackageVersion(ctx context.Context) error {
 	}
 
 	sc := snapshots.NewSnapshotsClient()
+
+	// TODO: find a proper fix.
+	if currentReleaseBranch == "8.x" {
+		currentReleaseBranch = "master"
+	}
```

Comment:
I think we should support for such branch in this function https://github.com/elastic/elastic-agent/blob/eb0f1300b42b9ad00df3ace4a3231a83d85edf84/pkg/testing/tools/snapshots/snapshots.go#L88-L122

The snapshot CDN should eventually have a manifest for 8.x (or perhaps different name?) and `master` will point to 9.0. I think it does not work now only because of the transitional period.

Reply:
> I think it does not work now only because of the transitional period.
agreed. I just wanna to unblock CI on this branch